xml/art_jeos_quickstart.xml

```diff
@@ -216,6 +216,34 @@
     </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>systemd-coredump disabled</term>
+     <listitem>
+      <para>
+       Because of the small amount of memory and disk space available by default on JeOS images,
+       systemd-coredump is disabled by default on JeOS images.
+       If your usage scenario requires to collect application core dumps for analyzing crashes:
```

Comment:
```suggestion
       If your usage scenario requires to collect application core dumps for analyzing crashes:
```

To collect application core dumps necessary for troubleshooting, follow these steps:

Reply:
Also, it's better to use a <procedure> here instead of <itemizedlist>